DIKU-DevStudio/PerseQ
# SNPCollection

**DONE**
- When adding a list of SNPs (for now hardcoded list of snps) user is notified of which snps were not formatted properly, and which were unknown to GWAS
- `CollectionList`: list of collections unique to logged in user
- `Collection`: view collection, listing the related SNPs

**TESTING**
- Add collection to a user by typing "/collection/<collection_name>" - if the collection does not exist, it is created.
- Add snps to a collection by clicking the button in the colletion (notice the _warning_ and _errors_)

**TODO**
- form for adding/deleting collections
- adding/deleting individual snps from a collection

@jensraaby if you feel like it, try adding the forms for addind a snp

[#27007257] initial collection
